server/tests/models/comment.spec.js

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable no-console */
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import chaiAsPromise from 'chai-as-promised';
+import models from '<serverModels>';
+import { getUserId, getArticleId } from '<test>/helpers/utils';
+
+chai.use(chaiAsPromise);
+
+const { expect } = chai;
+
+chai.use(chaiHttp);
+
+const { Comment, Article, sequelize } = models;
+
+before(async () => {
+  try {
+    await sequelize.sync({ force: true });
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+let userId;
+let articleId;
+let comment;
+let newComment;
```

Comment:
Please put these definitions before the hook

Reply:
Okay, 
I am guessing the reason is to write global variable declarations before the test.
I will do that now.
Thank you.